src/envs/pendulum.jl

```diff
@@ -7,14 +7,15 @@ using RecipesBase
 using Distributions
 using Random: seed!
 
-import Reinforce: reset!, actions, finished, step!
+import Reinforce: reset!, actions, finished, step!, state
```

Comment:
Oh, I still want this part of your PR. Could you reopen it?

Reply:
Sure, but I also added it to my new pull request...

Reply:
I think I actually can't because I recreated the branch for my new PR...but I added this suggestion as well...